pages/data.jsx

```diff
@@ -1,16 +1,101 @@
 import React from 'react'
+import { TextField } from '@oacore/design'
+
+import Table from '../components/table'
+import dataClassNames from './data.css'
+
+const range = len => {
+  const arr = []
+  for (let i = 0; i < len; i++) arr.push(i)
+
+  return arr
+}
```

Comment:
There is a functional way:
```js
const range = (length, start = 0, step = 1) => 
  Array.from(Array(length), (_, i) => start + i * step))
```

Also, you can use a generator:

```js
const range = len => {
  for (let i = 0; i < len; ++i) yield i
}
```

but to have an array you will need to do `Array.from(range(len))`